### DISCUSSION TITLE
why after build the website, gen the file resources/_gen/assets/scss/sass/main.scss_a4b227022dfae4b28843e276c7e6b126.content

### DISCUSSION THREAD

**Discussion by xtccc** [hugo-sid/hugo-blog-awesome#117] [Q&A]
## 
after I run the hugo 
it generate the `resources/_gen/assets/scss/sass/main.scss_a4b227022dfae4b28843e276c7e6b126.content`       `resources/_gen/assets/scss/sass/main.scss_a4b227022dfae4b28843e276c7e6b126.json`

It is a bug ? I don't want to  add the two file to the git 
```shell
hugo 

git status
public/   
resources/_gen/assets/scss/sass/main.scss_a4b227022dfae4b28843e276c7e6b126.content
resources/_gen/assets/scss/sass/main.scss_a4b227022dfae4b28843e276c7e6b126.json

```
hugo-blog-awesom verson is 
* main aa7630b [origin/main] ci: update Hugo version for Netlify builds

**Comment by hugo-sid** [comment]
> It is a bug ?

This is not a bug.

From https://gohugo.io/getting-started/directory-structure/#directories :
> The resources directory contains cached output from Hugo’s asset pipelines, generated when you run the hugo or hugo server commands. By default this cache directory includes CSS and images. Hugo recreates this directory and its content as needed.

Feel free to add this folder to your `.gitignore`.
